demos/interactive_face_detection_demo/main.cpp

```diff
@@ -211,10 +212,15 @@ int main(int argc, char *argv[]) {
 
         std::cout << "To close the application, press 'CTRL+C' here";
         if (!FLAGS_no_show) {
-            std::cout << " or switch to the output window and press any key";
+            std::cout << " or switch to the output window and press Q, q or Esc";
```

Comment:
```suggestion
            std::cout << " or switch to the output window and press Q or Esc";
```

Q and q are the same key. I don't think it makes sense to differentiate between them.

Reply:
Updated.